test/Ansible.common

```diff
@@ -1,16 +1,20 @@
 ANSIBLE_FOLDER ?= $(CURDIR)
-REQUIREMENTS_FILE ?= $(ANSIBLE_FOLDER)/requirements.yml
-
-ROLES_PATH:=$(ANSIBLE_FOLDER)/roles
-COLLECTIONS_PATH:=$(ANSIBLE_FOLDER)/collections
+export REQUIREMENTS_FILE ?= $(ANSIBLE_FOLDER)/requirements.yml
+export ROLES_PATH:=$(ANSIBLE_FOLDER)/roles
+export COLLECTIONS_PATH:=$(ANSIBLE_FOLDER)/collections
+export CROWDSTRIKE_ROLE_PULL_KEY := $(HOME)/.ssh/crowdstrike_ansible_role_key
```

Comment:
Is this variable used outside the provision directory?

Reply:
It's used in the script along with the Ansible-related ones (roles path, collections path, reqs file...)